lib/File/ShareDir.pm

```
@@ -435,6 +435,13 @@ else
 {
     ## no critic (ErrorHandling::RequireCheckingReturnValueOfEval)
     eval <<'END_OF_BORROWED_CODE';
+
```

NOPE. This is internal and must not be used from outside.

---

Is there any way hide this error?

---

Yes, make sure you have List::MoreUtils 0.428 installed.

---

Indeed it hides the error.

---

Which error. Can you show me the error?
I've tested it with Perls which have no LMU 0.428 installed and they don't show any error

---

The author POD test fail with a `firstres` is undocumented error. There is an eval 'use LMU 0.428' in the lines above it (line 430) which then imports it.

---

Oh, there were two errors in the description.

Yeah - for that failure, ignoring the 'firstres' in the test would be right.